### DISCUSSION TITLE
担忧

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2591] [其他]
这个绳网要开发APP版本了，入门条件不知道会不会变简单，很担心那些水军，黑子，机器人冲进来啊<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by lllliufeng** [comment]
应该还是要Github账号吧，而且手机在Github编辑帖子更麻烦吧（而且油猴插件不好搞定），所以变化应该不大

**Reply by JobNumber-9527** [reply]
担忧，看看up怎么处理吧唉

**Comment by JobNumber-9527** [comment]
如果要用Github账号注册才能登录的话就好了<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >
